example/src/account/account.resolver.ts

```diff
@@ -0,0 +1,17 @@
+import { Resolver, Args, Query, ID } from "@nestjs/graphql";
+import * as DataLoader from "dataloader";
+import { Loader } from "../../..";
+import { Account } from "./account.entity";
+import { AccountLoader } from "./account.loader";
+
+@Resolver("Account")
+export class AccountResolver {
+  @Query(() => [Account])
+  public getAccounts(
+    @Args({ name: "ids", type: () => [ID] }) ids: string[],
+    @Loader(AccountLoader.name)
+    accountLoader: DataLoader<Account["id"], Account>
+  ): Promise<(Account | Error)[]> {
```

Comment:
return `: Promse<(Account | Error>[]` ? 

https://github.com/krislefeber/nestjs-dataloader/blob/master/README.md#using-the-nestdataloader